### PR TITLE
[ODS-5584] Making SSL mount optional for AdminApp image

### PR DESCRIPTION
**Pull request by FluentGuru**
**Context**

Right now Ed-Fi Docker images are hardwired to only run if they have an SSL certificate mounted in a docker volume. This makes it incompatible to run in a cloud environment where SSL encryption is usually handled by another service like a load balancer or an API Gateway. An example of this behavior can be seen in the link below.

https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Docker/blob/5ae5208f3dde940ae14e5fc93a79323419e77b21/Web-Ods-AdminApp/Alpine/pgsql/run.sh#L44

**Solution**

Modify Ed-Fi Docker images so that the SSL mount is optional

### CODE REVIEW

**Comment by stephenfuqua** [comment]
@FluentGuru just curious - why did you close this PR?

@pmcvtm can you summarize the internal discussion on how we'll handle this change?

**Comment by FluentGuru** [comment]
> @FluentGuru just curious - why did you close this PR?
> 
> @pmcvtm can you summarize the internal discussion on how we'll handle this change?

Hi @stephenfuqua I closed the PR cause there was an issue with a commit that I couldn't amend to verify so after much trying I decided to just recreated the PR. Here's the new one
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Docker/pull/127
